docs/host/host.md

```diff
@@ -38,7 +38,7 @@ If you want to execute a special command before shutting down a VM, consider usi
 
 ## Cordoning a Node
 
-Cordoning a node marks it as unschedulable. This feature is useful for performing short tasks on the node during small maintenance windows, like reboots, upgrades, or decommissions. When you’re done, power back on and make the node schedulable again by uncordoning it.
+Cordoning a node marks it as unschedulable. This feature is useful when you don't want new workload to be scheduled to a node. You can make the node schedulable again by uncordoning it.
```

Comment:
```suggestion
Cordoned nodes are marked as unschedulable. Cordoning is useful when you want to prevent new workloads from being scheduled on a node. You can uncordon a node to make it schedulable again.
```